org-gcal.el

```diff
@@ -591,7 +592,7 @@ TO.  Instead an empty string is returned."
                       end
                     (org-gcal--iso-previous-day end)))))) "\n"
 		    (when desc "\n")
-		    desc
+		    (replace-regexp-in-string "\*" "✱" desc)
```

Comment:
This needs to be `(when desc (replace-regexp-in-string "\*" "✱" desc))` in order to work. (I didn't test if the outgoing function works)